containers/html/alert.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/containers/css/alert.css">
+    <title>alert | Components</title>
+</head>
+<body>
+    <div class="alert-containers">
+        <div class="alert">
+            <p class="text-in-alert">Here is some text!</p>
+            <button class="btn-in-alert">x</button>
+        </div>
+        <div class="alert success-div">
+            <p class="text-in-alert">Here is some text!</p>
+            <button class="btn-in-alert success-btn">x</button>
```

Comment:
Try to use a standard naming convention like BEM so that it is easy to understand.